input/fsh/instances/bulk-data.fsh

```diff
@@ -0,0 +1,46 @@
+Instance: bulk-data
+InstanceOf: CapabilityStatement
+Usage: #definition
+* url = "http://hl7.org/fhir/uv/bulkdata/CapabilityStatement/bulk-data"
+* version = "2.0.0"
+* name = "BulkDataIGCapabilityStatement"
+* title = "FHIR Bulk Data Access Implementation Guide"
+* status = #active
+* experimental = false
+* date = "2021-07-29"
+* description = "The expected capabilities of a Bulk Data Provider actor (e.g., EHR systems, data warehouses, and other clinical and administrative systems that aim to interoperate by sharing large FHIR datasets) which is responsible for providing responses to the queries submitted by a FHIR Bulk Data Client actor. Systems implementing this capability statement should meet the requirements set by the Bulk Data Access Implementation Guide. A FHIR Bulk Data Client has the option of choosing from this list to access necessary data based on use cases and other contextual requirements."
+* jurisdiction = $m49.htm#001 "World"
+* kind = #requirements
+* instantiates = "http://hl7.org/fhir/uv/bulkdata/CapabilityStatement/bulk-data"
+* fhirVersion = #4.0.1
+* format = #json
+* implementationGuide = "http://hl7.org/fhir/uv/bulkdata/ImplementationGuide/hl7.fhir.uv.bulkdata"
+* rest
+  * mode = #server
```

Comment:
You may find it easier to use the triple quote syntax for these multi-lines strings with markdown in them..

https://github.com/HL7/cgm/blob/3754690f8b7a6534b7ead9a84ae9ce08043746b3/input/fsh/profiles.fsh#L580 Is an example

Reply:
Neat, thanks! I'll move to triple quotes as I edit long strings (right now they're a straight copy and paste from the json versions of these resources).